public/js/services/UserService.js

```diff
@@ -0,0 +1,90 @@
+﻿app.factory('UserService', ['$http', function ($http) {
+  
+  var factory = {};
+  
+  factory.getAll = function () {
+    return $http.get('/users')
+        .success(function (data) {
```

Comment:
Angular has deprecated .success and .error promises. They should be replaced with .then(successFunction, errorFunction)
https://docs.angularjs.org/api/ng/service/$http#deprecation-notice